README.md

```diff
@@ -140,25 +140,33 @@ watt_watt_container
 mongo                                                                             3.4.19              056cb4b05c15        5 weeks ago         376MB
 ```
  * [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-linux.html)
- * [Push images to aws repositories](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.html#use-ecr).
- * If you need to update WATT image only, just tag your local watt image with Uri from [repositories](https://us-east-2.console.aws.amazon.com/ecr/repositories?region=us-east-2) and push the image (no need to create separate image repositories). Pushing mongod image can be also omitted.
- * Images should be available at [repositories](https://us-east-2.console.aws.amazon.com/ecr/repositories?region=us-east-2).
+ * Push watt and mongo images to [aws repositories](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.html#use-ecr) to desired region. To push the image to [Asia Pacific Seoul](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) you can pass --region ap-northeast-2 while creating repository..
+ * If you need to update WATT image only, just tag your local watt image with Uri from [repositories](https://us-east-2.console.aws.amazon.com/ecr/repositories?region=ap-northeast-2) and push the image.
+ * Images should be available at [repositories](https://us-east-2.console.aws.amazon.com/ecr/repositories?region==ap-northeast-2).
 
-## Setup WATT on AWS
- * Update docker-compose-aws.yml with new docker images repositories.
- * You can change default memory limits for each container in ecs-params.yml
- * [Install the Amazon ECS CLI](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI_installation.html)
- * [Create Your Cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-cli-tutorial-ec2.html), to prepare a separate instance. You can omitt this steep in order to use already existing watt cluster.
+## Creating cluster
+ * You can steeps below in order to use already existing watt cluster.
+ * [Install the Amazon ECS CLI](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI_installation.html).
+ * Create network infrastructure:
+    * Define [VPC with Elastic IP](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-public-private-vpc.html).
+    * Define *security group* using previously created *VPC*.
+    * Add 22 and 3000 ports to *Inboud Roules* of newly created security group.
+ * [Create Your Cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-cli-tutorial-ec2.html), for example, 
 ```bash
```

Comment:
Can we use in README some variable like REGION
which can be described as:
us-east-2
ap-northeast-2

instead of changing us-east-2 to ap-northeast-2 in a whole guide?

Reply:
I changed the whole guide to ap-northeast-2 since the most cases of aws creation will be dedicated for HQ (Asia). If some wants to change the region it's easy configurable by params.

More beneficial in this PR is how to set network infrastructure for new cluster.